docs/index.asciidoc

```diff
@@ -72,6 +72,13 @@ Events will be tagged with `_geoip_expired_database` tag to facilitate the handl
 
 TIP: When possible, allow Logstash to access the internet to download databases so that they are always up-to-date.
 
+**Disable the auto-update feature**
+
+If you work in air-gapped environment, and want to disable the auto-update feature from Logstash, you can do so by setting the `xpack.geoip.db.auto_update`
+configuration to `false` in `logstash.yml`.
+
+When disabled, a valid `database` location is required, and you are responsible for <<plugins-{type}s-{plugin}-manage_update,manage your own database updates>> and <<plugins-{type}s-{plugin}-database_license,maintaining compliance>>.
```

Comment:
```suggestion
When automatic updates are disabled, a valid `database` location is required.
You are responsible for managing your own database updates and <<plugins-{type}s-{plugin}-database_license,maintaining license compliance>>.
```

Reply:
Consider removing the link to "maintaining compliance."  It's the very next topic and clicking the link doesn't really take you anywhere.